sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/validation/RegionValidator.java

```diff
@@ -0,0 +1,70 @@
+package com.mesosphere.sdk.specification.validation;
+
+import com.mesosphere.sdk.config.validate.ConfigValidationError;
+import com.mesosphere.sdk.offer.evaluate.placement.PlacementRule;
+import com.mesosphere.sdk.offer.evaluate.placement.PlacementUtils;
+import com.mesosphere.sdk.specification.PodSpec;
+import com.mesosphere.sdk.specification.ServiceSpec;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * This class validates that referencing Regions in placement constraints can only support the following transitions.
+ *
+ * 1. null  --> false
```

Comment:
It is not clear how the `null` to `false` transition is validated. (and by extension that `null` to `true` is *not* allowed.